lib/active_fedora/associations/builder/association.rb

```diff
@@ -67,18 +63,6 @@ def self.better_name(name)
       name
     end
 
-    def self.translate_property_to_predicate(options)
-      return unless options[:property]
-      Deprecation.warn Association, "the :property option to `#{model}.#{macro} :#{name}' is deprecated and will be removed in active-fedora 10.0. Use :predicate instead", caller(5)
-      options[:predicate] = predicate(options.delete(:property))
-    end
-
-    # Returns the RDF predicate as defined by the :property attribute
-    def self.predicate(property)
```

Comment:
Yes, it's only used by a deprecated method.  We didn't add a deprecation to that method too, otherwise the user would get two deprecation warnings when they did the deprecated behavior.